README.md

```diff
@@ -6,3 +6,43 @@ These are the documents and resources to be expected here 2016:
 * Point symbols in SVG format and as a symbol font (TTF).
 * Config. file (osmaxx.json) to be listed as "Project" in taginfo web app
 * Other related stuff
+
+
+## Font Generation
+
+The font-generation for OSMaxx can be found in `fontforge_font_creator`.
+
```

Comment:
We should probably make clear that "the font" in this section and its subsections is a (map) (point) _symbol_ font, not a font for writing or for labels.

Suggestion: Change section title to "Symbol Font Generation"